tests/plugins/test_flatpak_create_oci.py

```diff
@@ -475,7 +475,7 @@ def build_export(repo, directory, branch):
       '--oci', '@filename', '@name', '@branch'],
      MockFlatpak.build_bundle),
     # For build-export, we assume the latest flatpak version for flatpak_module_tools compatibility
-    (['flatpak', 'build-export', '@repo', '@directory', '@branch', '--disable-sandbox'],
```

Comment:
I did reproduce the failure locally and reverting this change will fix the broken test.

See discussion #1160 as reference for why we need this flag.

About reproducing the error locally: It seems this tests are being skipped even using the same script as the CI does. I needed to install krb5-devel and python-koji in the container to get it to run. Maybe some package got updated in Fedora and didn't reach CI mirrors yet?

Reply:
I added that change to get my local tests to work.  That is so crazy.  Well, I deleted that commit and things are green again.